app/reducers/encryption.ts

```diff
@@ -1,11 +1,18 @@
 import { ENCRYPTION } from '../actions/actionsTypes';
+import { TApplicationActions } from '../definitions';
 
-const initialState = {
+export interface IEncryption {
+	enabled: boolean;
+	// TODO
+	banner: any;
```

Comment:
I think banner could be a string.

Reply:
I even put a TODO because I wasn't sure what it was, I'll put string.